lms/resources/_js_config/__init__.py

```diff
@@ -273,6 +274,8 @@ def enable_dashboard_mode(self):
                 ),
             }
         )
+        if user_info := self._get_user_info():
```

Comment:
Make this always present there's no scenario where this would be empty.

Or mark the type as "NotRequired" if we prefer that.